test/onnx/onnx_import.in.cpp

```diff
@@ -0,0 +1,1412 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <algorithm>
+#include <cmath>
+#include <cstdint>
+#include <fstream>
+#include <iterator>
+#include <limits>
+#include <sstream>
+#include <stdexcept>
+#include <vector>
+
+#include "gtest/gtest.h"
+#include "ngraph/frontend/onnx_import/onnx.hpp"
+#include "ngraph/ngraph.hpp"
+#include "util/all_close.hpp"
+#include "util/all_close_f.hpp"
+#include "util/ndarray.hpp"
+#include "util/test_control.hpp"
+#include "util/test_tools.hpp"
+
+using namespace ngraph;
+
+static std::string s_manifest = "${MANIFEST}";
+
+using Inputs = std::vector<std::vector<float>>;
+using Outputs = std::vector<std::vector<float>>;
+
+// ############################################################################ CORE TESTS
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_output_names_check)
+{
+    auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/split_equal_parts_default.prototxt"));
+
+    std::size_t size = function->get_output_size();
+    for (std::size_t i{0}; i < size; ++i)
+    {
+        std::shared_ptr<Node> node = function->get_output_op(i);
+        EXPECT_EQ(node->get_friendly_name(), "output_" + std::to_string(i + 1));
+    }
+}
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_add_abc)
```

Comment:
Why add_abc, binary_add_abc and add_abc_initializers tests are in the core section but addmul_abc is in operators section?

Reply:
These tests were actually designed to test core functionality, so I grouped them there.